src/common/Screen.js

```diff
@@ -28,13 +28,14 @@ const componentStyles = StyleSheet.create({
 });
 
 type Props = {
-  padding?: boolean,
-  search?: boolean,
+  children: ChildrenArray<*>,
   centerContent: boolean,
   safeAreaInsets: Dimensions,
   scrollView: boolean,
+  keyboardShouldPersistTaps?: string,
+  padding?: boolean,
+  search?: boolean,
```

Comment:
Why reorder the props in this list that aren't changing?

Unless there's a clear reason the new order is better and canonical, let's leave it in place -- that avoids making the diff longer and more confusing to read.

Reply:
(Same for the defaults and the destructuring below. Except in those, it looks like you're putting them in alpha order? That's reasonable -- in that case, please
 * do the same here
 * mention in the commit message that that's what you're doing, to make it easier for the reader :)

)

Reply:
Yup! after almost working 1 year on this project, I realised that if they are ordered in alphabetically then it lot easier to read and re-use that component.
Also following the order: required props followed by optional and functions at the end of section. With this order it looks visually better too 😅 

Okay! will create separate commit for that.